### DISCUSSION TITLE
Easy to use menu

### DISCUSSION THREAD

**Discussion by nqtuyen** [oqtane/oqtane.framework#1469] [Ideas]
Hi Shaun Walker,
Thank for your great work.
My opinion for menu as 2 option below:
Option 1:
1. change edit button to This Page dropdown button with 2 command: edit content (same icon) & page settings
and move Page management & Module management section in Control panel to page settings. Collect all command for page to this.
2. Admin button will open Admin Dashboard directly.
Option 2:
1. when click edit button, append top toolbar with Page management button and toolbar in bottom Admin Pane with Insert module to page button (will open Module dialog for input parameter)
2. Admin button will open Admin Dashboard directly.
I think option 2 is more easer to understand and use for end user.
Please take a look and give me your opinion.
Thank you!
TuyenNguyen

**Comment by sbwalker** [comment]
@nqtuyen you are describing a couple of different solutions... but you did not explain the problem you are trying to solve so it is very difficult to follow the logic.
